### PR TITLE
GLTFExporter: Added onlyVisible option

### PR DESCRIPTION
**Pull request by fernandojsg**
I forgot to include this changes in a previous PR :angel:

### CODE REVIEW

File: examples/js/exporters/GLTFExporter.js [hunk 1]
```diff
@@ -782,13 +782,18 @@ THREE.GLTFExporter.prototype = {
...
 
 					var child = object.children[ i ];
-					if ( child instanceof THREE.Mesh ||
-						child instanceof THREE.Camera ||
-						child instanceof THREE.Group ||
-						child instanceof THREE.Line ||
-						child instanceof THREE.Points) {
 
-						children.push( processNode( child ) );
+					if ( !options.onlyVisible || child.visible ) {
```
Comment (mrdoob) [inline examples/js/exporters/GLTFExporter.js]: Maybe this is easier to read?

```js
if ( child.visible || options.onlyVisible === false ) {
```

**Comment by mrdoob** [comment]
Actually. What's the use case for exporting invisible objects?

File: examples/js/exporters/GLTFExporter.js (hunk 1 above)
Comment (fernandojsg) [inline examples/js/exporters/GLTFExporter.js]: Fixed!

**Comment by fernandojsg** [comment]
@mrdoob sometimes you would like to export all the meshes on the scene even if they're hidden, for example you could have an editor where you hide the background to focus on few objects and you don't want to toggle visibility of everything just to export, similar to how 3dsmax or similar works.

**Comment by fernandojsg** [comment]
but I agree that won't be the most common use case, that's why `onlyVisible` is set to `true` by default

**Comment by mrdoob** [comment]
Wouldn't it be cleaner to use `object.layers` for that?

**Comment by mrdoob** [comment]
Or even another scene. In https://threejs.org/editor/ I use another scene for the helpers.

**Comment by fernandojsg** [comment]
`Layers` could be another use case yep, but I still feel using just `visible` for an object could be useful. In `aframe-inspector` we allow the users to toggle the visibility on the sidebar:
image
If you're moving things around maybe you want to hide big objects that prevent you to work on the ones you want to focus so you just toggle few of them, but you still want to export the whole scene, and don't want to deal with layers and so on, as it's just part of your creation workflow but you won't use that visibility options on your final scene.
In the inspector I also use a different scene just for the helpers so it won't conflict with this option either.

**Comment by mrdoob** [comment]
I see I see 👌

**Comment by mrdoob** [comment]
Thanks!

**Comment by fernandojsg** [comment]
thanks! ;)

**Pull request merged**